docs/developers/spark-devel-env.rst

```diff
@@ -1,5 +1,5 @@
-Set up ListenBrainz Spark development environment
-=================================================
+ListenBrainz Spark development
+==============================
```

Comment:
1 line sentence saying what the spark environment is for?

The ListenBrainz Spark environment is used for computing statistics and computing recommendations

Reply:
Maybe we don't need the 3 lines below this either? Just a 1 sentence introduction, and then the comment that says that you don't need this if you're just developing the website